public/index.html

```diff
@@ -22,6 +22,9 @@
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
     <title>Aragon Court Dashboard</title>
+    <style>
+      body img {user-select: none;}
```

Comment:
I think we probably want to use styled-component's global stylesheet functionality instead—not sure if we already use it in this app.

Reply:
Ah wasn't aware of that feature 👍